spec/controllers/hyku/registrations_controller_spec.rb

```diff
@@ -0,0 +1,35 @@
+# frozen_string_literal: true
+
+RSpec.describe Hyku::RegistrationsController, type: :controller do
+  let(:account) { create(:account) }
+  let(:site) { Site.new(account: account) }
+
+  before do
+    allow(Site).to receive(:instance).and_return(site)
+
+    @request.env["devise.mapping"] = Devise.mappings[:user] # rubocop:disable RSpec/InstanceVariable
+  end
+
+  context "with account signup enabled" do
```

Comment:
There should be a context where account signup is disabled and we can't create accounts.

Reply:
@thenapking Those scenarios are covered inside of Hyku :)

Reply:
@thenapking Those scenarios are covered inside of Hyku :)

Reply:
Oh year, I see that bit did not change